### PR TITLE
add facebook share button

### PR DESCRIPTION
**Pull request by JohnL17**

### CODE REVIEW

File: app/core/templates/core/profile/description.html [hunk 1]
```diff
@@ -71,6 +71,14 @@ <h2 class="mt0">{{ politician.first_name }} {{ politician.last_name }}</h2>
             </div>
         </div>
         {% endif %}
+        <div class="social-media-buttons">
+            <div class="facebook">
+                <a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.freedomvote.ch%2Fpolitician%2F{{politician.id}}%2F" target="_blank" title="{% trans "Share" %}" role="button" aria-label="{% trans "Share" %}" class="share-button">
```
Comment (andreabettich) [inline app/core/templates/core/profile/description.html]: Is it possible to remove the hardcoded freedomvote.ch and replace it with the `BASE_URL`?

File: app/core/templates/core/profile/index.html [hunk 2]
```diff
@@ -16,3 +16,11 @@
     <hr>
     {% include 'core/profile/questions.html' %}
 {% endblock %}
+
+{% block 'meta' %}
+    <meta property="og:title" content="OSS-Challenge" />
+    <meta property="og:type" content="profile:username" />
+    <meta property="og:image" content="{{ politician.image }}" />
+    <meta property="og:url" content="www.freedomvote.ch/politician/{{ politician.id }}/" />
```
Comment (andreabettich) [inline app/core/templates/core/profile/index.html]: Same as above.

File: app/core/templates/core/profile/description.html (hunk 1 above)
Comment (anehx) [reply app/core/templates/core/profile/description.html]: @JohnL17 Yes please do that!

Comment (JohnL17) [reply app/core/templates/core/profile/description.html]: @andreabettich Yes of course. Thank you.

File: app/core/templates/core/profile/index.html [hunk 3]
```diff
@@ -16,3 +16,11 @@
     <hr>
     {% include 'core/profile/questions.html' %}
 {% endblock %}
+
+{% block 'meta' %}
+    <meta property="og:title" content="OSS-Challenge" />
```
Comment (anehx) [inline app/core/templates/core/profile/index.html]: This is most likely not the title you want..

File: app/core/templates/core/profile/index.html [hunk 4]
```diff
@@ -16,3 +16,11 @@
...
+    <meta property="og:type" content="profile:username" />
```
Comment (anehx) [inline app/core/templates/core/profile/index.html]: To my understanding `content` should be "website"

File: app/core/templates/core/profile/index.html [hunk 5]
```diff
@@ -16,3 +16,11 @@
...
+    <meta property="og:image" content="{{politician.image}}" />
+    <meta property="og:url" content="{{politician_url}}/" />
+    <meta property="og:description" content="{{politician.first_name}} {{politician.last_name}}" />
```
Comment (anehx) [inline app/core/templates/core/profile/index.html]: Maybe add something like "Hans Muster on Freedomvote" here

File: app/core/templates/core/profile/index.html (hunk 3 above)
Comment (JohnL17) [reply app/core/templates/core/profile/index.html]: Oops my fault, sorry.

File: app/core/templates/core/profile/index.html [hunk 6]
```diff
@@ -16,3 +16,11 @@
     <hr>
     {% include 'core/profile/questions.html' %}
 {% endblock %}
+
+{% block 'meta' %}
+    <meta property="og:title" content="{{politician.first_name}} {{politician.last_name}}" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="{{politician.image}}" />
+    <meta property="og:url" content="{{politician_url}}/" />
+    <meta property="og:description" content="The profile of {{politician.first_name}} {{politician.last_name}} on Freedomvote" />
```
Comment (anehx) [inline app/core/templates/core/profile/index.html]: This has to be multilingual!

File: app/core/templates/core/profile/index.html [hunk 7]
```diff
@@ -16,3 +16,11 @@
     <hr>
     {% include 'core/profile/questions.html' %}
 {% endblock %}
+
+{% block 'meta' %}
+    <meta property="og:title" content="{{politician.first_name}} {{politician.last_name}}" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="{{politician.image}}" />
```
Comment (andreabettich) [inline app/core/templates/core/profile/index.html]: Wouldn't this return the ImageField?
`{{politician.image.url}}` should work but will return the url without the baseurl.

**Comment by andreabettich** [comment]
Looks fine for me.

**Comment by anehx** [comment]
@JohnL17 Translations are still missing. Otherwise LGTM :+1:

**Comment by anehx** [comment]
Did you test it on mobile? Does it look OK?

**Pull request merged**